usr/src/uts/common/brand/lx/procfs/lx_prvnops.c

```diff
@@ -4923,7 +4928,23 @@ lxpr_read_sys_kernel_pid_max(lxpr_node_t *lxpnp, lxpr_uiobuf_t *uiobuf)
 	lxpr_uiobuf_printf(uiobuf, "%d\n", maxpid);
 }
 
-/* ARGSUSED */
+static void
+lxpr_gen_uuid(char *uuid, size_t size)
+{
+	uint8_t r[16];
+	(void) random_get_bytes(r, sizeof (r));
```

Comment:
`random_get_bytes()` can possibly fail if there is insufficient entropy, returning EAGAIN.
In that case, the UUID is going to be built from whatever was on the stack.
I think we should rather just fall back to `random_get_pseudo_bytes()` so it's at least a bit more random.

```suggestion
	if (random_get_bytes(r, sizeof (r)) != 0)
	        (void) random_get_pseudo_bytes(r, sizeof (r));
```

Reply:
Thanks for the feedback, falsely assumed it wouldn't fail as the original code didn't check the return value either.